src/style.css

```diff
@@ -1,3 +1,65 @@
-body {
+:root {
+  --blue-color: #00acdc; /* Определение переменной для синего цвета */
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-align: left;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Don't use the `*` selector as it impacts performance. Set styles only for elements that require them.